todo_app/app.py

```diff
@@ -7,50 +8,98 @@
 from flask_dance.contrib.github import github
 from werkzeug.middleware.proxy_fix import ProxyFix
 
+import logging, os, logging.config, time, loggly.handlers
+from loggly.handlers import HTTPSHandler
+from logging import Formatter
+# logging.config.fileConfig('python.conf')
+# logging.config.fileConfig(r'C:\DevOps\M1.ProjectExercise\DevOps-Course-Starter-v2\todo_app\python.conf')
+logging.Formatter.converter = time.gmtime
+logger = logging.getLogger(__name__)
+# logger = logging.getLogger('myLogger')
+
+# logger.info('Test INFO')
+# logger.debug('Test DEBUG')
+# logger.warning('Test WARNING')
+# logger.error('Test ERROR')
+# logger.critical('Test CRITICAL')
```

Comment:
It's good practice to clean up these kind of debug lines before merging your code, simply for readability/code cleanliness.